packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -1,21 +1,48 @@
 import {api, LightningElement} from 'lwc';
 
 import inLabel from "@salesforce/label/c.quantic_InLabel";
+
+/** @typedef {import("coveo").CategoryFacetValue} CategoryFacetValue */
+
+/**
+ * The `QuanticCategoryFacetValue` component is used by a `QuanticCategoryFacet` component to display a formatted facet value, path to that value and the number of results with that value.
+ * @fires CustomEvent#selectvalue
+ * @example
+ * <c-quantic-category-facet-value onselectvalue={onSelect} item={result} is-search-result active-parent></c-quantic-category-facet-value>
+ */
 export default class QuanticCategoryFacetValue extends LightningElement {
-  /** @type {import("coveo").CategoryFacetValue} */
+  /**
+   * The [facet value](https://docs.coveo.com/en/headless/latest/reference/controllers/category-facet/#categoryfacetvalue) to display.
+   * @api
+   * @type {CategoryFacetValue} */
   @api item;
-  /** @type {string} */
-  @api withsearch;
-  /**  @type {string} */
-  @api activeparent;
-  /**  @type {boolean}*/
-  @api nonactiveparent;
+  /**
+   * Whether the value is a search result.
```

Comment:
IMO `search result` can easily be confused with the documents returned by the platform. Maybe using `facet search result` would be clearer.